components/textbook-beta/StartLearningSection.vue

```diff
@@ -168,3 +184,19 @@ export default class StartLearningSection extends Vue {
   }
 }
 </style>
+
+<style lang="scss">
+// override
+.start-learning-section {
+  .app-link {
+    text-decoration: none;
+    color: $link-color;
+    &:hover {
+      color: $link-hover-color;
+    }
+    &:visited {
+      color: $link-visited-color;
+    }
+  }
+}
```

Comment:
Why are we overriding this?

The current color (blue) is the one we defined for the "app links". It can be seen in the homepage for instance.

Personally, I believe that we should either keep them all as currently defined or change them all (for example to purple), but not create a special case here.

I would like to hear what @JRussellHuffman thinks about this. 🙂 

To add some visual help, here is how it would look without this special override (meaning, as we defined the links should look like):

<img width="837" alt="Screenshot 2021-09-15 at 18 11 14" src="https://user-images.githubusercontent.com/22047320/133470069-d73ccba3-ccc0-48be-88c7-7b922c16f5a0.png">

And here is how it looks with this override:

<img width="975" alt="Screenshot 2021-09-15 at 18 12 44" src="https://user-images.githubusercontent.com/22047320/133470151-89dbafd0-f99d-4b0b-8fd2-2878d8e8c2f0.png">

Personally, I like using purple instead of blue, since it feels more on-brand. But we should do so for all links IMO. Also, the shade of purple –again, IMO– is quite hard to be distinguished from regular text. I'd like to see either more contrast or an underline, or some kind of highlighting in general.

Reply:
Thanks for pointing this out, @eddybrando 

I was confused here because in the design, it shows **no** underline:

<img width="1189" alt="Screen Shot 2021-09-15 at 10 16 07 AM" src="https://user-images.githubusercontent.com/6276074/133479425-26263001-ab72-4ccb-a205-5335a55d2811.png">

So I went and looked for this style, that has the primary blue but no underline... and found it on the https://qiskit.org site, under "Start locally"

so I tried reusing its classes, to achieve the same style... however, I got the underline 🤔 

this is why I was trying to override.

Nonetheless, I removed the overriding styles... 👍🏽

Reply:
Apologies for the confusion. I agree with Eddy that we should stick with the default link styling.

Not related to the link, but the text: The copy has been updated very slightly. One sentence is removed. This should be the last edit on the text, but if it changes again, I will let you all know.